grammars/Kotlin.tmLanguage.json

```
@@ -303,7 +303,7 @@
             "name": "entity.name.type.annotation.kotlin"
         },
         "class-declaration": {
-            "match": "\\b(class|interface)\\s+(\\b\\w+\\b|`[^`]+`)\\s*(?<GROUP><([^<>]|\\g<GROUP>)+>)?",
+            "match": "\\b(class|interface|fun\\s+interface)\\s+(\\b\\w+\\b|`[^`]+`)\\s*(?<GROUP><([^<>]|\\g<GROUP>)+>)?",
```

Just an idea, haven't checked if it's valid, but could this be rephrased to

```suggestion
            "match": "\\b(class|(?:fun\\s+)?interface)\\s+(\\b\\w+\\b|`[^`]+`)\\s*(?<GROUP><([^<>]|\\g<GROUP>)+>)?",
```

---

Seems to work, thanks!